.github/workflows/release-package.yml

```diff
@@ -0,0 +1,33 @@
+# Publish a new package when a pre-release PR is merged.
+#
+# Job does nothing if PR that was merged is not a pre-release PR.
+
+name: "Publish @webref package if needed"
+
+on:
+  pull_requests:
+    branches:
+      - 'release-*'
+    types: [closed]
+
+jobs:
+  release:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14.x
+
+    - name: Checkout webref
+      uses: actions/checkout@v2
```

Comment:
Will this check out the branch of the PR that was closed? Or a merge commit of that and main, as usually happens with PRs?

Reply:
That's unrelated to the PR. This will checkout the latest commit on the main branch. This checkout is only needed to be able to run the latest version of the `release-package.js` script.

Reply:
Sorry, I just realized that my previous comment is totally wrong. The [checkout action](https://github.com/actions/checkout#checkout-v2) will checkout the ref/sha that triggered the workflow. In this case, this will be the [last merge commit on the PR branch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request).

In turn, this means that possible fixes/changes made to the release script after a pre-release PR has been created would not be taken into account when the PR is merged. That seems counter-intuitive and not a good thing. I'll update the job to checkout the main branch instead.